app/assets/javascripts/incident_reports/edit.coffee

```diff
@@ -54,17 +54,36 @@ deleteInjuredPassengerFields = (event) ->
   deleteFields '.pax-fields', '#supervisor_report_inj_pax_info'
 
 toggleReasonsForTesting = ->
-  reason = $('#supervisor_report_reason_test_completed').val()
-  switch reason
-    when 'Post-Accident'
-      $('.post-accident-info').slideDown()
-      $('.reasonable-suspicion-info').slideUp()
-    when 'Reasonable Suspicion'
-      $('.post-accident-info').slideUp()
-      $('.reasonable-suspicion-info').slideDown()
-    when ''
-      $('.post-accident-info').slideUp()
-      $('.reasonable-suspicion-info').slideUp()
+  reason = $('#supervisor_report_test_status').val()
+  if reason.includes('Reasonable Suspicion')
+    $('.post-accident-info').slideUp()
+    $('.reasonable-suspicion-info').slideDown()
+    $('.driver-discounted-info').slideUp()
+    $('.fta-threshold-info').slideUp()
+
+  else if reason.includes('Threshold met (completed drug test)')
+    $('.driver-discounted-info').slideUp()
+    $('.reasonable-suspicion-info').slideUp()
+    $('.fta-threshold-info').slideUp()
+    $('.post-accident-info').slideDown()
+
+  else if reason.includes('No threshold met')
+    $('.driver-discounted-info').slideUp()
+    $('.fta-threshold-info').slideDown()
+    $('.post-accident-info').slideUp()
+    $('.reasonable-suspicion-info').slideUp()
+
+  else if reason.includes('discounted')
+    $('.driver-discounted-info').slideDown()
+    $('.fta-threshold-info').slideUp()
+    $('.post-accident-info').slideUp()
+    $('.reasonable-suspicion-info').slideUp()
+
+  else
+    $('.driver-discounted-info').slideUp()
+    $('.fta-threshold-info').slideUp()
+    $('.post-accident-info').slideUp()
+    $('.reasonable-suspicion-info').slideUp()
```

Comment:
More kind-of-nitpicks, but I think perhaps abstracting at least all the slideup/down stuff to other methods would be good.

Reply:
Yep, alright

Reply:
Done.